lib/jxl/icc_codec.cc

```diff
@@ -181,33 +181,33 @@ Status UnpredictICC(const uint8_t* enc, size_t size, PaddedBytes* result) {
         tagstart = prevtagstart + prevtagsize;
       }
       JXL_RETURN_IF_ERROR(CheckIs32Bit(tagstart));
-      AppendUint32(tagstart, result);
+      JXL_RETURN_IF_ERROR(AppendUint32(tagstart, result));
       if (command & kFlagBitSize) {
         if (cpos >= commands_end) return JXL_FAILURE("Out of bounds");
         tagsize = DecodeVarInt(enc, size, &cpos);
       }
       JXL_RETURN_IF_ERROR(CheckIs32Bit(tagsize));
-      AppendUint32(tagsize, result);
+      JXL_RETURN_IF_ERROR(AppendUint32(tagsize, result));
       prevtagstart = tagstart;
       prevtagsize = tagsize;
 
       if (tagcode == kCommandTagTRC) {
-        AppendKeyword(kGtrcTag, result);
-        AppendUint32(tagstart, result);
-        AppendUint32(tagsize, result);
-        AppendKeyword(kBtrcTag, result);
-        AppendUint32(tagstart, result);
-        AppendUint32(tagsize, result);
+        JXL_RETURN_IF_ERROR(AppendKeyword(kGtrcTag, result));
+        JXL_RETURN_IF_ERROR(AppendUint32(tagstart, result));
+        JXL_RETURN_IF_ERROR(AppendUint32(tagsize, result));
+        JXL_RETURN_IF_ERROR(AppendKeyword(kBtrcTag, result));
+        JXL_RETURN_IF_ERROR(AppendUint32(tagstart, result));
+        JXL_RETURN_IF_ERROR(AppendUint32(tagsize, result));
       }
 
       if (tagcode == kCommandTagXYZ) {
         JXL_RETURN_IF_ERROR(CheckIs32Bit(tagstart + tagsize * 2));
-        AppendKeyword(kGxyzTag, result);
-        AppendUint32(tagstart + tagsize, result);
-        AppendUint32(tagsize, result);
-        AppendKeyword(kBxyzTag, result);
-        AppendUint32(tagstart + tagsize * 2, result);
-        AppendUint32(tagsize, result);
+        JXL_QUIET_RETURN_IF_ERROR(AppendKeyword(kGxyzTag, result));
```

Comment:
still QUIET?

Reply:
oops, I though Sami had fixed it..

Should be fixed now